package/system/ca-certificates/files/update-ca-certificates

```diff
@@ -0,0 +1,214 @@
+#!/bin/sh -e
```

Comment:
Could we please use tabs a not spaces?
This would reduce the size of the file

Reply:
I can, but if we ever wanted to resync this with debian sources it would be awkward